pipeline/matcher_merger/merger/src/main/scala/weco/pipeline/merger/MergerSQSLambda.scala

```diff
@@ -0,0 +1,57 @@
+package weco.pipeline.merger
+
+import weco.lambda.{
+  ApplicationConfig,
+  Downstream,
+  SQSBatchResponseLambdaApp,
+  SQSLambdaMessage,
+  SQSLambdaMessageFailedRetryable,
+  SQSLambdaMessageResult
+}
+import weco.pipeline.merger.services.WorkRouter
+import weco.pipeline.merger.Main.WorkOrImage
+import scala.concurrent.Future
+import scala.util.Try
+
+trait MergerSQSLambda[Config <: ApplicationConfig]
+    extends SQSBatchResponseLambdaApp[String, Config] {
```

Comment:
The merger receives a `MatcherResult` and we have a method for extracting this at the top level in `SQSEventOps`.  Like:

```
 extends SQSBatchResponseLambdaApp[MatcherResult, Config]
```

This is preferable to re-implementing the decoding logic in this lambda.